README.markdown

```diff
@@ -296,6 +307,36 @@ steps to create a new migration script for h are:
    $ alembic -c conf/alembic.ini upgrade +1
    ```
 
+### Installing Dev Application In Canvas
+
+1. Start your dev server as described above.
+
+2. Navigate to `<dev-url>/welcome` in your browser.
+
+3. Provide your canvas domain and email in the form.
+After submitting the form you should be provided with a
+consumer key and secret. Make note of these.
+
+4. Navigate to the settings of a course in canvas.
+
+5. Navigate to the Apps tab, then the App Configurations.
+
+6. Add an application, and select the 'Paste XML' option from the dropdown.
+
+7. Copy your consumer key and secret into the canvas form you received
+previously from the welcome page.
+
+8. Navigate to `<dev-url>/config_xml` in your browser.
```

Comment:
I personally had trouble with this with Firefox. I'm not sure if it would be worth mentioning some troubleshooting tips here or not.

Reply:
I think I will add that if we hear another of another person having trouble.